File: examples/sorting-algorithms/case-reverse-sorted.ts
const data = Array.from({ length: 1000 }, (_, i) => 999 - i);

export function createCase() {
  return {
    case: {
      caseId: "reverse-sorted",
      caseClass: "sorted",
      name: "Reverse Sorted Array",
      version: "1.0.0",
      inputs: { data },
    },
    getInput: async () => ({ data }),
    getInputs: () => ({ data }),
  };
}

File: examples/sorting-algorithms/case_reverse_sorted.py
"""Reverse Sorted Array case — [999..0]."""

_data = list(range(999, -1, -1))


def create_case():
    return {
        "case": {
            "caseId": "reverse-sorted",
            "caseClass": "sorted",
            "name": "Reverse Sorted Array",
            "version": "1.0.0",
            "inputs": {"data": _data},
        },
        "getInput": lambda: {"data": _data},
        "getInputs": lambda: {"data": _data},
    }
